src/devices/Bmxx80/Bme280.cs

```diff
@@ -141,5 +169,25 @@ private double CompensateHumidity(int adcHumidity)
 
             return varH;
         }
+
+        private bool TryReadHumidityInternal(out double humidity, bool skipTempFineRead = false)
+        {
+            if (HumiditySampling == Sampling.Skipped)
+            {
+                humidity = double.NaN;
+                return false;
+            }
+
+            // Read the temperature first to load the t_fine value for compensation.
+            if (!skipTempFineRead)
+            {
+                TryReadTemperatureInternal(out _);
```

Comment:
should you not be ignoring result?

Reply:
See https://github.com/dotnet/iot/pull/797#discussion_r432845583